public/js/team/HeroView.js

```diff
@@ -1,6 +1,6 @@
 /* globals Backbone */
 /* exported HeroView */
```

Comment:
I'm starting to think these comments were put in place for jshint instead of eslint. Everything in the public folder is read by the browser and not Node, so some of these changes need to be reverted. If there's no eslint rule to allow certain unused variables, than these should be set to `window.HeroView`, for example, instead.

Reply:
I apologize for not catching this earlier. I should have more closely examined the lint errors.